### DISCUSSION TITLE
Inbound and Outbound traffic

### DISCUSSION THREAD

**Discussion by BlueWS** [tnodir/fort#283] [Q&A]
Hello there,
just installed Fort Firewall today to try it out and i like the interface and the options available.
As an old Malwarebytes Windows Firewall Control i used to make separate applications rules for inbound and outbound traffic so that i could block both or one of them.
So i wanted to create a rule that blocks the outbound traffic but i couldn't find how to do that in the 'Rules' panel.
So i'm wondering is that possible as of now or not implemented/planned?
On the main Fort Firewall guy i saw no column for In/Out which also made me wonder if it's possible to do that.
Thanks for reading and eventual answers!
T.

**Comment by tnodir** [comment]
Hello.
It's not implemented yet.

**Reply by BlueWS** [reply]
Hi tnodir, thanks for the reply!
Happy to hear that cause it means that it *will* be added at some point later on.
Looking forward to it beside future features!
